resources/definitions/trimaker_nebula_plus.def.json

```diff
@@ -0,0 +1,249 @@
+            "generic_pp",
+            "generic_pva",
+            "generic_pva_175",
+            "generic_tough_pla",
+            "generic_tpu",
+            "generic_tpu_175",
+            "imade3d_petg_175",
+            "imade3d_pla_175",
+            "innofill_innoflex60_175",
+            "leapfrog_abs_natural",
+            "leapfrog_epla_natural",
+            "leapfrog_pva_natural",
+            "octofiber_pla",
+            "polyflex_pla",
+            "polymax_pla",
+            "polyplus_pla",
+            "polywood_pla",
+            "structur3d_dap100silicone",
+            "tizyx_abs",
+            "tizyx_flex",
+            "tizyx_petg",
+            "tizyx_pla",
+            "tizyx_pla_bois",
+            "tizyx_pva",
+            "ultimaker_abs_black",
+            "ultimaker_abs_blue",
+            "ultimaker_abs_green",
+            "ultimaker_abs_grey",
+            "ultimaker_abs_orange",
+            "ultimaker_abs_pearl-gold",
+            "ultimaker_abs_red",
+            "ultimaker_abs_silver-metallic",
+            "ultimaker_abs_white",
+            "ultimaker_abs_yellow",
+            "ultimaker_bam",
+            "ultimaker_cpe_black",
+            "ultimaker_cpe_blue",
+            "ultimaker_cpe_dark-grey",
+            "ultimaker_cpe_green",
+            "ultimaker_cpe_light-grey",
+            "ultimaker_cpe_plus_black",
+            "ultimaker_cpe_plus_transparent",
+            "ultimaker_cpe_plus_white",
+            "ultimaker_cpe_red",
+            "ultimaker_cpe_transparent",
+            "ultimaker_cpe_white",
+            "ultimaker_cpe_yellow",
+            "ultimaker_nylon_black",
+            "ultimaker_nylon_transparent",
+            "ultimaker_pc_black",
+            "ultimaker_pc_transparent",
+            "ultimaker_pc_white",
+            "ultimaker_pla_black",
+            "ultimaker_pla_blue",
+            "ultimaker_pla_green",
+            "ultimaker_pla_magenta",
+            "ultimaker_pla_orange",
+            "ultimaker_pla_pearl-white",
+            "ultimaker_pla_red",
+            "ultimaker_pla_silver-metallic",
+            "ultimaker_pla_transparent",
+            "ultimaker_pla_white",
+            "ultimaker_pla_yellow",
+            "ultimaker_pp_transparent",
+            "ultimaker_pva",
+            "ultimaker_tough_pla_black",
+            "ultimaker_tough_pla_green",
+            "ultimaker_tough_pla_red",
+            "ultimaker_tough_pla_white",
+            "ultimaker_tpu_black",
+            "ultimaker_tpu_blue",
+            "ultimaker_tpu_red",
+            "ultimaker_tpu_white",
+            "verbatim_bvoh_175",
+            "Vertex_Delta_ABS",
+            "Vertex_Delta_PET",
+            "Vertex_Delta_PLA",
+            "Vertex_Delta_PLA_Glitter",
+            "Vertex_Delta_PLA_Mat",
+            "Vertex_Delta_PLA_Satin",
+            "Vertex_Delta_PLA_Wood",
+            "Vertex_Delta_TPU",
+            "zyyx_pro_flex",
+            "zyyx_pro_pla"

```

Comment:
These are a lot of excluded materials. Have you tested all of these, and found that they don't work? We consider blindly excluding third party materials to be quite anti-competitive so please only exclude materials that are provably infeasible.

And I wonder how you tested them, since some of them are 1.75mm diameter and some of them are 2.85mm. Materials that don't have the correct filament diameter are automatically excluded by Cura and don't need to be in this list.